src/views/Scenario/Scenario.js

```diff
@@ -288,6 +305,73 @@ const Scenario = () => {
     label: scenarioListLabel,
     validationStatus: scenarioValidationStatusLabels,
   };
+
+  const userPermissionsOnCurrentWorkspace = workspace?.data?.security?.currentUserPermissions || [];
```

Comment:
add a useUserPermissionOnCurrentWorkspace in a WorkSpaceHooks and use a useMemo for refresh issue